src/prompt_toolkit/filters/app.py

```diff
@@ -170,6 +171,33 @@ def has_validation_error() -> bool:
     return get_app().current_buffer.validation_error is not None
 
 
+@Condition
+def has_any_validation_error() -> bool:
+    "Any buffer in the layout has validation error."
+    from prompt_toolkit.layout.containers import Window
+    from prompt_toolkit.layout.controls import (
+        BufferControl,
+        SearchBufferControl,
+        UIControl,
+    )
+
+    # Extract buffer validation errors for buffer UIControl children classes
+    def get_buffer_from_content(content: UIControl) -> bool:
```

Comment:
Could you rename this? This doesn't return the buffer. For instance `buffer_has_validation_error()`